source/uwp/AdaptiveCardsObjectModel/AdaptiveCardsObjectModel.vcxproj

```diff
@@ -43,13 +43,15 @@
     <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
     <AppContainerApplication>true</AppContainerApplication>
     <ApplicationType>Windows Store</ApplicationType>
-    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
-    <WindowsTargetPlatformMinVersion>10.0.15063.0</WindowsTargetPlatformMinVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```

Comment:
Is this change OK? I was being lazy and updated these to make VS happy; not sure this is what you want.

Reply:
Visual Studio seems happy enough without it on my end, so reverting this change.